README.md

```diff
@@ -346,6 +346,16 @@ Don't become a daemon and execute on foreground on the associated terminal.
 
 The path of the PID file; the default is */var/run/puppetlabs/pxp-agent.pid*
 
+## Maintenance
+
+Maintainers: Alessandro Parisi <alessandro@puppet.com>, Michael Smith
+<michael.smith@puppet.com>, Michal Ruzicka <michal.ruzicka@puppetlabs.com>.
```

Comment:
uh, oh: `<michal.ruzicka@puppetlabs.com>` -> `<michal.ruzicka@puppet.com>`
sorry for that, didn't notice it in the HipChat